core/cloudflow-operator/src/test/scala/cloudflow/operator/TestDeploymentContext.scala

```diff
@@ -22,8 +22,8 @@ trait TestDeploymentContext {
   implicit val ctx: DeploymentContext =
```

Comment:
Should this get a better name to express it relies on Strimzi?

Reply:
Do you mean DeploymentContext relies on strimzi? I'm not sure what you mean?

Reply:
The DeploymentContext is fine without strimzi, the user then just can't use the 'managed topic feature'.

Reply:
(There is more work needed after this PR..)